src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Badge/AssignmentList.html

```diff
@@ -15,6 +15,7 @@ <h2 class="badges-group-header">{{badgeGroups[groupPath].title}}</h2>
                         class="badge-checkbox"
                         data-ng-model="checkboxes[badgePath]">
                         {{badges[badgePath].title}}
+                    </input>
```

Comment:
Inputs are void elements, so the do not have a closing tag. The XHTML issue here is that it needs to be closed with `/>` instead of just `>`. Also, the indentation is misleading becuase it suggests that `{{badges[badgePath].title}}` is a child node of `<input>`.